Tests/scripts/pkg_dev_test_tasks.py

```diff
@@ -260,6 +261,16 @@ def run_mypy(project_dir, py_num):
     print("mypy completed")
 
 
+def run_bandit(project_dir, py_num):
+    lint_files = get_lint_files(project_dir)
+    print("========= Running bandit on: {} ===============".format(lint_files))
+    python_exe = 'python2' if py_num < 3 else 'python3'
+    print_v('Using: {} to run flake8'.format(python_exe))
+    sys.stdout.flush()
+    subprocess.check_call([python_exe, '-m', 'bandit', '-lll', '-iii', lint_files], cwd=CONTENT_DIR)
```

Comment:
Let's add `-q`. It prints out a lot of stuff when there is no need.
```suggestion
    subprocess.check_call([python_exe, '-m', 'bandit', '-lll', '-iii', '-q', lint_files], cwd=CONTENT_DIR)
```